bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2ThingHandler.java

```diff
@@ -154,11 +155,13 @@ public class Clip2ThingHandler extends BaseThingHandler {
     private final ThingRegistry thingRegistry;
     private final ItemChannelLinkRegistry itemChannelLinkRegistry;
     private final Clip2StateDescriptionProvider stateDescriptionProvider;
+    private final TimeZoneProvider timeZoneProvider;
 
     private String resourceId = "?";
     private Resource thisResource;
     private Duration dynamicsDuration = Duration.ZERO;
     private Instant dynamicsExpireTime = Instant.MIN;
+    private Instant lastUpdated = Instant.MIN;
```

Comment:
Could you please rename this as `buttonGroupLastUpdated`

Reply:
Renamed.

Reply:
Many thanks. My reason was not to confuse the last updated channel at button (group) level with that at device level. Also the motion/light/temperature sensors should NOT use it, nor the forthcoming security contact and tamper services. And I am pretty sure that even the rotary dial does not require it.